core/src/main/java/org/apache/carbondata/core/locks/LocalFileLock.java

```diff
@@ -70,7 +71,8 @@
       LogServiceFactory.getLogService(LocalFileLock.class.getName());
 
   static {
-    tmpPath = System.getProperty("java.io.tmpdir");
+    tmpPath = CarbonProperties.getInstance().getProperty(CarbonCommonConstants.STORE_LOCATION,
+        System.getProperty("java.io.tmpdir"));
```

Comment:
Still that code is not yet merged. it is present in https://github.com/apache/incubator-carbondata/pull/736